rllib/tuned_examples/qmix/two-step-game-qmix-no-mixer.yaml

```diff
@@ -0,0 +1,22 @@
+two-step-game-qmix-without-mixer:
+    env: ray.rllib.examples.env.two_step_game.TwoStepGameWithGroupedAgents
+    run: QMIX
+    stop:
+        episode_reward_mean: 8.0
+        timesteps_total: 70000
+    config:
+        # QMIX only supports torch for now.
+        framework: torch
```

Comment:
try
```
frameworks: ["torch"] 
```
at the top level to only run these tests for torch?
I see that these new tests are failing because of framework tf.